client/src/views/uploadPage.vue

```diff
@@ -204,9 +183,12 @@ async function uploadCroppedImage() {
         <input
           type="file"
           ref="lunarImage"
-          accept="images/*"
+          accept="image/*"
```

Comment:
using html's api to enforce file type is not reliable. I can still select non-image files.
![image](https://github.com/Gavin1937/Your-Moon-CSULA/assets/71205842/75fa1438-8602-4015-b4b2-4b4ca1631475)